tests/test_gear_generator.py

```diff
@@ -0,0 +1,78 @@
+import cadquery as cq
+from plugins.gear_generator.gear_generator import *
+from unittest import TestCase
+from math import sin, radians
+
+class TestGearGenerator(TestCase):
+    def test_bevel_gear(self):
+        """
+        Tests if the bevel gear has been created successfully 
+        by checking if it's valid and it's resulting volume
+        Numerical values volumes were obtained with the Volume() functions on a Windows 10 machine with python 3.9.2
```

Comment:
These tests are failing on Ubuntu 18.04. In two cases it looks like you can just reduce the number of decimal places to check. For this test, the following is what I see.
```
>       self.assertAlmostEqual(gear1.val().Volume(),2505.7531046471067)
E       AssertionError: 2505.7966225164664 != 2505.7531046471067 within 7 places (0.04351786935967539 difference)
```